docker-ci.sh

```diff
@@ -195,6 +195,13 @@ function run_test_scss {
     npm i && npm run build && npm run test
 }
 
+function run_test_license {
+    npm i
+    npm run build
+
+    npx gulp check-license-notices
```

Comment:
This task is called twice: https://devextreme-ci.devexpress.com/DevExpress/DevExtreme/34035/51